platform/saivpp/vpplib/SwitchStateBaseRif.cpp

```diff
@@ -44,6 +44,33 @@ using namespace saivpp;
 
 int SwitchStateBase::currentMaxInstance = 0;
 
+/* Utility function for IP addr translation from VPP to SAI */
+static void vpp_to_sai_ip_addr(
```

Comment:
Let's move it to the central place SwitchStateUtils.h to be used by other modules